applications/mne_anonymize/main.cpp

```diff
@@ -40,8 +40,8 @@
 // INCLUDES
 //=============================================================================================================
 
-#include "settingscontroller.h"
-
+#include "settingscontrollerCL.h"
```

Comment:
`settingscontrollerCL.h` should be `settingscontrollercl.h` captial letters in file names are discouraged.

Reply:
done